packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -66,5 +66,8 @@
   "experimental_deploy_retrieve_description": "Enable performance enhancements to the deploy/retrieve experience.",
   "experimental_apex_library_description": "Use Apex Library for Apex CLI commands",
   "force_function_create": "SFDX: Create Function",
+  "force_function_start": "SFDX: Start Function",
```

Comment:
@ekapner Command names for local functions. Please let me know if this looks good. This won't be exposed until we flip the flag.

Reply:
@danyusf - These looks good to me; and follow the command names and verbiage you use in the Dev Guide.

Reply:
LGTM (assuming this basically wraps the evergreen:function:start cmd)

Reply:
Thanks, yeah it's a wrapper over the plugin command